tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -1,51 +1,202 @@
 import logging
 
+import boto3
+import botocore
 import pytest
 
 from ocs_ci.framework.pytest_customization.marks import (
-    tier1, noobaa_cli_required, aws_platform_required, acceptance,
-    filter_insecure_request_warning
+    tier1, tier2, tier3, noobaa_cli_required, aws_platform_required,
+    acceptance, filter_insecure_request_warning
 )
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.mcg_bucket import S3Bucket, OCBucket, CLIBucket
+from ocs_ci.utility.utils import run_mcg_cmd
+from tests.helpers import craft_s3_command, create_unique_resource_name
 
 logger = logging.getLogger(__name__)
 
 
 @filter_insecure_request_warning
 @aws_platform_required
-@acceptance
-@tier1
 class TestBucketDeletion:
     """
     Test bucket Creation Deletion of buckets
     """
-    @pytest.mark.polarion_id("OCS-1299")
-    def test_s3_bucket_delete(self, mcg_obj, bucket_factory):
+    @pytest.mark.parametrize(
+        argnames="amount,interface",
+        argvalues=[
+            pytest.param(
+                *[3, 'S3'],
+                marks=[pytest.mark.polarion_id("OCS-1299"), tier1, acceptance]
+            ),
+            pytest.param(
```

Comment:
I think it is redundant to execute all sizes over all the different APIs. Let's focus on OC for all sizes and for s3 and CLI, test only the functional (amount=3)

Reply:
CLI + OC might be a bit redundant, but S3 is very important IMO. It's an entirely different control plane, and one that'll most definitely be used by client.
I suggest we do OC+S3 for all sizes, and keep CLI functional-only.